src/compiler/compiler.rs

```diff
@@ -1401,10 +1398,7 @@ LLVM version: 6.0",
             }))
             .unwrap();
         // Ensure that the object file was created.
-        assert_eq!(
-            true,
-            fs::metadata(&obj).and_then(|m| Ok(m.len() > 0)).unwrap()
-        );
+        assert_eq!(true, fs::metadata(&obj).map(|m| m.len() > 0).unwrap());
```

Comment:
We should rewrite this as an `assert_eq(Ok(1),...)`

Reply:
I didn't get it, it's Result(Integer) x bool